src/orion/algo/mofa/sampler.py

```diff
@@ -0,0 +1,119 @@
+"""
+MOFA Sampler module
+"""
+
+# from skopt.sampler import Lhs
+# from skopt.space import Space as SkSpace
+
+import logging
+
+import numpy as np
+from scipy.stats.qmc import LatinHypercube
+
+logger = logging.getLogger(__name__)
+
+
+class Sampler:
+    """
+    Samples trial parameters from an orthogonal Latin hypercube.
+    """
+
+    def __init__(self):
+        return
+
+    @staticmethod
+    def _is_prime(number):
+        if number < 2 or (number % 2 == 0 and number > 2):
```

Comment:
2 is prime, and as far as I can tell, this will not work, but instead raise an error with the `range(3, 2, 2)` below.
How about something a bit more explicit:
```suggestion
        if number == 2:
            return True
        if number < 2 or number % 2 == 0:
            return False
```

Reply:
Nevermind, range(3,2,2) doesn't raise an error, it's just an empty iterable, and so this doesn't go through the for loop, it just returns `True`, as it should.